components/common/src/main/java/com/hotels/styx/common/FsmEventProcessor.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (C) 2013-2017 Expedia Inc.
+ * Copyright (C) 2013-2018 Expedia Inc.
```

Comment:
My intellij started to do that too, 2018, does it matter from license point of view?